doc/fuzzing.md

```diff
@@ -153,13 +153,18 @@ You may also need to take care of giving the correct path for `clang` and
 `clang++`, like `CC=/path/to/clang CXX=/path/to/clang++` if the non-systems
 `clang` does not come first in your path.
 
+Currently the default linker on macOS `ld` does not work with recent versions of `llvm` so you will need to install llvm's linker `lld` and use it with the CXX_FLAG `-fuse-ld=lld`.
+
 Full configuration step that was tested on macOS with `brew` installed `llvm`:
```

Comment:
This seems to need an update - though I'd just remove the command from the comment:
```suggestion
Full configuration steps:
```

Reply:
Took something similar